app/uk/gov/hmrc/cataloguefrontend/service/LeakDetectionService.scala

```diff
@@ -36,26 +36,12 @@ class LeakDetectionService @Inject()(leakDetectionConnector: LeakDetectionConnec
       }
     }
 
-  private val leakDetectionPublicUrl = {
-    val key = "lds.publicUrl"
-    configuration
-      .getString(key)
-      .getOrElse(
-        throw new Exception(s"Failed reading from config, expected to find: $key")
-      )
-  }
+  private val leakDetectionPublicUrl: String = configuration.get[String]("lds.publicUrl")
 
-  private val ldsIntegrationEnabled: Boolean = {
-    val key = "lds.integrationEnabled"
-    configuration
-      .getBoolean(key)
-      .getOrElse(
-        throw new Exception(s"Failed reading from config, expected to find: $key")
-      )
-  }
+  private val ldsIntegrationEnabled: Boolean = configuration.get[Boolean]("lds.integrationEnabled")
 
   val repositoriesToIgnore: List[String] =
-    configuration.getStringList("lds.noWarningsOn").fold(List.empty[String])(_.asScala.toList)
+    configuration.underlying.getStringList("lds.noWarningsOn").asScala.toList
```

Comment:
Maybe a good candidate to add to the Bootstrap an implicit ConfigLoader for List[String]?